### PR TITLE
Fix ambient occlusion stacking

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Fixes filter not being removed and relays stacking up

## Changelog

:cl:
fix: fixed ambient occlusion stacking sometimes
/:cl:

### CODE REVIEW

File: code/_onclick/hud/rendering/plane_master.dm [hunk 1]
```diff
@@ -13,6 +13,8 @@
 	var/generate_render_target = TRUE
 	///integer: blend mode to apply to the render relay in case you dont want to use the plane_masters blend_mode
 	var/blend_mode_override
+	///reference: current relay this plane is utilizing to render
+	var/atom/movable/render_plane_relay/relay
```
Comment (Ghilker) [inline code/_onclick/hud/rendering/plane_master.dm:17]: where does this gets cleared?

Comment (Watermelon914) [reply code/_onclick/hud/rendering/plane_master.dm:17]: likely when the client disconnects I'd assume. Nothing else holds a reference to it and it gets autocleared by the game's GC.

Comment (Watermelon914) [reply code/_onclick/hud/rendering/plane_master.dm:17]: Assuming this gets deleted when the client gets deleted.

Comment (Ghilker) [reply code/_onclick/hud/rendering/plane_master.dm:17]: Wouldn't be better to null it before having to GC it? Not making any assumptions here I'm just curious

Comment (Watermelon914) [reply code/_onclick/hud/rendering/plane_master.dm:17]: If you look at circuits, this is what they did before and it was unnecessary and ended up clogging the files a lot. You don't have to qdel or destroy something that'll get gc'd on its own

**Pull request merged**